pkg/cmd/roachtest/testdata/pg_regress/create_index.diffs

```diff
@@ -2569,12 +2569,15 @@ diff -U3 --label=/mnt/data1/postgres/src/test/regress/expected/create_index.out
  DROP TABLE cwi_test;
  --
  -- Check handling of indexes on system columns
-@@ -1610,20 +1541,46 @@
+@@ -1610,20 +1541,49 @@
  ERROR:  relation "syscolcol_table" does not exist
  -- nor used in expressions
  CREATE INDEX ON syscol_table ((ctid >= '(1000,0)'));
 -ERROR:  index creation on system columns is not supported
-+ERROR:  column "ctid" does not exist, referenced in "ctid >= '(1000,0)'"
++ERROR:  this schema change is disallowed because table "syscol_table" is locked and this operation cannot automatically unlock the table
++DETAIL:  To unlock the table, execute `ALTER TABLE syscol_table SET (schema_locked = false);`
++After the schema change completes, we recommend setting it back to true with `ALTER TABLE syscol_table SET (schema_locked = true);`.
++HINT:  Locking the table improves changefeed performance; see https://www.cockroachlabs.com/docs/_version_/changefeed-best-practices.html#lock-the-schema-on-changefeed-watched-tables
```

Comment:
Should we unset `create_table_with_schema_locked` for pg_regress?

Reply:
Good idea. I tried this and looks like `pg_regress` gets stuck now in the middle of all tests. I'll take another closer look later this week.